src/main/java/mongodb/Main.java

```diff
@@ -0,0 +1,220 @@
+import org.bson.conversions.Bson;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.mongodb.client.model.Filters.*;
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+public class Main {
+  public static void main(String[] args) {
+    // TODO:
+    // Replace the placeholder connection string below with your
+    // Altas cluster specifics. Be sure it includes
+    // a valid username and password! Note that in a production environment,
+    // you do not want to store your password in plain-text here.
+    ConnectionString mongoUri = new ConnectionString("<Your Atlas Connection String>");
+
+    // Provide the name of the database and collection you want to use.
+    // If they don't already exist, the driver and Atlas will create them
+    // automatically when you first write data.
+    String dbName = "myDatabase";
+    String collectionName = "recipes";
+
+    // a CodecRegistry tells the Driver how to move data between Java POJOs (Plain Old Java Objects) and MongoDB documents
+    CodecRegistry pojoCodecRegistry = fromRegistries(MongoClientSettings.getDefaultCodecRegistry(),
+            fromProviders(PojoCodecProvider.builder().automatic(true).build()));
+
+    // The MongoClient defines the connection to our MongoDB datastore instance (Atlas) using MongoClientSettings
+    MongoClientSettings settings = MongoClientSettings.builder()
+            .codecRegistry(pojoCodecRegistry)
+            .applyConnectionString(mongoUri).build();
+    MongoClient mongoClient = MongoClients.create(settings);
+
+    // MongoDatabase defines a connection to a specific MongoDB database
+    MongoDatabase database = mongoClient.getDatabase(dbName);
+    // MongoCollection defines a connection to a specific collection of documents in a specific database
+    MongoCollection<Recipe> collection = database.getCollection(collectionName, Recipe.class);
+
+    /*      *** INSERT DOCUMENTS ***
+     *
+     * You can insert individual documents using collection.insert().
+     * In this example, we're going to create 4 documents and then
+     * insert them all in one call with insertMany().
+     */
+
+    List<Recipe> recipes = getRecipes();
+
+    try {
+      InsertManyResult result = collection.insertMany(recipes);
+      System.out.println("Inserted document ids: " + result.getInsertedIds());
+    } catch (MongoException me) {
+      System.err.println("Unable to insert any driver documents due to an error: " + me);
+    }
+
+    /*      *** FIND DOCUMENTS ***
+     *
+     * Now that we have data in Atlas, we can read it. To retrieve all of
+     * the data in a collection, we call find() with an empty filter.
+     */
+
+    try (MongoCursor<Recipe> cursor = collection.find().iterator()) {
+      while (cursor.hasNext()) {
+        System.out.println(cursor.next());
```

Comment:
Question: this is different than Caleb's output which I modeled my example after. Should we conform to the script? It would also give you a chance to use the getters in the POJO: https://github.com/mongodb-university/atlas_starter_nodejs/pull/1/files#diff-0364f57fbff2fabbe941ed20c328ef1aR117